static/scss/_variables.scss

```diff
@@ -8,3 +8,10 @@ $micromasters-brand-fg: #2BA853;
 $fg-grey: #F5F5F5;
 $border-color: #cccccc;
 $search-filter: #D2E4F9;
+$progress-widget-bg-color: #ddd;
+$progress-widget-fg-color: #2ecc40;
```

Comment:
@roberthouse54 should the colors for the progress widget be the same as our overall `micromasters-brand` colors?